config.py

```diff
@@ -56,17 +59,20 @@ class Config(object):
     ACCESS_CASE = 'surveys/access-case'
     DOWNLOAD_CI = 'surveys/download-ci'
     UPLOAD_CI = 'surveys/upload-ci'
+    ADD_SURVEY = 'surveys/add-survey'
+    CONFIRM_ADD_ORGANISATION_SURVEY = 'surveys/add-survey/confirm-add-organisation-survey'
 
 
 class DevelopmentConfig(Config):
     DEVELOPMENT = True
     DEBUG = True
     TEMPLATES_AUTO_RELOAD = True
     SECRET_KEY = os.getenv('SECRET_KEY', 'ONS_DUMMY_KEY')
-    JWT_SECRET = os.getenv('JWT_SECRET', 'vrwgLNWEffe45thh545yuby')
+    JWT_SECRET = os.getenv('JWT_SECRET', 'testsecret')
     SECURITY_USER_NAME = os.getenv('SECURITY_USER_NAME', 'test_user')
     SECURITY_USER_PASSWORD = os.getenv('SECURITY_USER_PASSWORD', 'test_password')
     BASIC_AUTH = (SECURITY_USER_NAME, SECURITY_USER_PASSWORD)
+    NEW_SURVEY_NOTIF_HIGHLIGHTING = os.getenv('NEW_SURVEY_NOTIF_HIGHLIGHTING', '1')
```

Comment:
You don't need this both here and in config. It's only if it's needed but not in config or the default is different. As Development config inherits from the config object where it is already defined.